web/gui/dashboard_info.js

```diff
@@ -849,7 +849,37 @@ netdataDashboard.submenu = {
     'apps.net': {
         title: 'network',
         info: 'Netdata also gives a summary for eBPF charts in <a href="#menu_ip_submenu_kernel">Networking Stack submenu</a>.'
-    }
+    },
+
+    'system.ipc semaphores': {
+        info: 'System V semaphores is an inter-process communication (IPC) mechanism that widely available on UNIX systems. '+
+        'It allows processes or threads within a process to synchronize their actions. '+
+        'A semaphore is a value in a designated place in operating system (or kernel) storage that each process can check and then change. '+
+        'Depending on the value that is found, the process can use the resource or will find that it is already in use and must wait for some period before trying again. '+
+        'They are often used to monitor and control the availability of system resources such as shared memory segments. ' +
+        'For details, see <a href="https://man7.org/linux/man-pages/man7/svipc.7.html">svipc(7)</a> and '+
+        '<a href="https://man7.org/linux/man-pages/man7/sem_overview.7.html">sem_overview(7)</a>. ' +
+        'To see the host IPC semaphore information, run <code>ipcs -us</code>. For limits, run <code>ipcs -ls</code>.'
+    },
+
+    'system.ipc shared memory': {
+        info: 'System V shared memory is an inter-process communication (IPC) mechanism that widely available on UNIX systems. '+
+        'It allows allows processes to communicate information by sharing a region of memory. '+
+        'It is the fastest form of inter-process communication available since no kernel involvement occurs when data is passed between the processes (no copying). '+
```

Comment:
Lines `868` and `858` are long lines creating horizontal bar on my window, considering the other lines we have for the dashboard, I think this is not expected.

Reply:
> creating horizontal bar on my window

Could you show? There are many much longer lines in the dashboard_info.js.

Reply:
Of course, see : 
![Screenshot_20210916_191847](https://user-images.githubusercontent.com/49162938/133672441-c9e2e913-f41f-47af-831d-a92ae28530cc.png)

I agree that `dashboard_info.js` has a lot of lines like this, but I ping you about these lines, because they were the unique long.

Reply:
I don't really follow, there are lines like

https://github.com/netdata/netdata/blob/6a3e230174533ae0772a29714a12a983f2318584/web/gui/dashboard_info.js#L1203 **(1096 characters)**

So there is no way to get rid of "horizontal bar on my window" until we have _fixed_ all of them.

Reply:
I see you mean when reviewing using GitHub UI. Didn't catch it because I use "split" view which has no such problem

<img width="1388" alt="Screenshot 2021-09-16 at 22 30 02" src="https://user-images.githubusercontent.com/22274335/133674179-b4413bb4-5140-48fd-b410-757fdaf75beb.png">

Reply:
I will address the `eBPF` descriptions that are making long lines soon, I am only finishing the integration with `cgroups` before to work with documentation and dashboard again.

Reply:
I'd prefer to not change it. It is (almost always) a line per sentence, if I change it it will be messy. 

And it depends on the font size. As you see to fit it for me using "unified" view I would shorten the lines even more.

<img width="1346" alt="Screenshot 2021-09-16 at 22 44 18" src="https://user-images.githubusercontent.com/22274335/133676011-7d4384a9-226e-4df1-a497-419930323696.png">
e)